addons/l10n_lt/data/account_chart_template_data.xml

```diff
@@ -0,0 +1,1495 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo noupdate="1">
+    <!-- Chart of Accounts Template -->
+    <record id="account_chart_template_lithuania" model="account.chart.template">
+        <field name="name">Lithuania - Accounting</field>
+        <field name="code_digits">1</field>
```

Comment:
why not 6 ?

Reply:
Users were not liking those leading zeros, so we used 1 to avoid it. Though, if this is more general use case, I can change to default one that is 6?

Reply:
alright then, keep it 1